src/Akeneo/Connectivity/Connection/front/src/settings/components/ConnectionEditForm.tsx

```diff
@@ -1,13 +1,17 @@
 import {useFormikContext} from 'formik';
 import React from 'react';
 import styled from 'styled-components';
-import {FormGroup, FormInput, InlineHelper, Section} from '../../common';
+import {Checkbox, FormGroup, FormInput, InlineHelper, Section} from '../../common';
 import {Connection} from '../../model/connection';
+import {FlowType} from '../../model/flow-type.enum';
 import {Translate} from '../../shared/translate';
 import {FormValues} from '../pages/EditConnection';
+import {AuditableHelper} from './AuditableHelper';
 import {FlowTypeHelper} from './FlowTypeHelper';
 import {FlowTypeSelect} from './FlowTypeSelect';
-import ImageUploader from './ImageUploader';
+import {ImageUploader} from './ImageUploader';
+
+const isAuditForbid = (flowType: FlowType) => flowType === FlowType.OTHER;
```

Comment:
```suggestion
const isAuditForbidden = (flowType: FlowType) => flowType === FlowType.OTHER;
```
I would say "Forbidden"